tools/py_common/btree_format.py

```diff
@@ -514,3 +517,97 @@ def prepared(self) -> bool:
         Check if this cell belongs to a prepared transaction.
         '''
         return self.extra_descriptor & Cell.WT_CELL_PREPARE != 0
+    
+    def descriptor_string(self) -> str:
+        desc_str = f'desc: 0x{self.descriptor:x} '
+        if self.extra_descriptor != 0:
+            desc_str += f'extra: 0x{self.extra_descriptor:x} '
+            # process_timestamps(p, cell, pagestats)
+        if self.run_length is not None:
+            desc_str += + f'runlength/addr: {binary_data.d_and_h(self.run_length)} '
+        
+        return desc_str
+    
+    def type_string(self) -> str:
+        type_str = '? unknown type'
+        if self.is_address:
+            type_str = 'addr (leaf no-overflow) '
+        elif self.is_key:
+            type_str = 'key '
+        elif self.is_value:
+            type_str = 'val '
+        elif self.is_unsupported:
+            type_str = f'celltype = {self.cell_type.value}, cellname = {self.cell_type.name} not implemented'
+            
+        if self.is_overflow:
+            type_str = 'overflow ' + type_str
+        if self.is_short:
+            type_str = 'short ' + type_str
```

Comment:
Use `f-strings` to be consistent:

```suggestion
            type_str = f'overflow {type_str}'
        if self.is_short:
            type_str = f'short {type_str}'
```

Reply:
fixed https://github.com/wiredtiger/wiredtiger/pull/12932/changes/89626f483b408b463384adf12c0d2192793704fc